src/mixxx.cpp

```diff
@@ -265,6 +265,13 @@ void MixxxMainWindow::initialize(QApplication* pApp, const CmdlineArgs& args) {
             ConfigKey("[Config]", "StartInFullscreen"));
     if (args.getStartInFullscreen() || fullscreenPref) {
         showFullScreen();
+#ifdef __LINUX__
+        // If the desktop features a global menubar and we go fullscreen during
+        // startup, move the menubar to the window like in slotViewFullScreen()
+        if (m_pMenuBar->isNativeMenuBar()) {
+            m_pMenuBar->setNativeMenuBar(false);
+        }
+#endif
```

Comment:
looks like this was already fixed in main by simply calling `slotViewFullScreen(true)`